homeassistant/components/plaato/__init__.py

```diff
@@ -100,15 +106,24 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
                 hass.config_entries.async_forward_entry_setup(entry, platform)
             )
 
-    entry.add_update_listener(async_reload_entry)
+    entry.add_update_listener(_async_update_listener)
```

Comment:
`add_update_listener` returns a callback that needs to be stored and called when the config entry is unloaded.

Reply:
Looks good 👍